src/lib/styles/global/variables.scss

```diff
@@ -17,6 +19,7 @@
   --padding-2x: calc(2 * var(--padding));
   --padding-3x: calc(3 * var(--padding));
   --padding-4x: calc(4 * var(--padding));
+  --padding-5x: calc(5 * var(--padding));
```

Comment:
is this a must as we use nowhere else such a spacing?

Reply:
Maybe @artkorotkikh-dfinity can help us.

It used 40px in the design. I didn't know we weren't using it anywhere and because it's a mulitple of 8 I didn't see any problem in adding it.

But I don't have a strong opinion in using 4x instead.

Reply:
Not a strong opinion neither but if it doesn't look bad with 4x I would stick to the common spacing for consistency reason but again, not a strong opinion neither.

Reply:
I will leave the decision to @artkorotkikh-dfinity 

I'll ask him